specs/spec_helper.js

```diff
@@ -0,0 +1,18 @@
+var shell = require('shelljs');
+
+const testSetup = () => {
+  shell.exec('npx sequelize db:create')
+  shell.exec('npx sequelize db:migrate')
+  shell.exec('npx sequelize db:seed:all')
+}
+
+const tearDown = () => {
+  shell.exec('npx sequelize db:seed:undo:all')
+  shell.exec('npx sequelize db:migrate:undo:all')
+  shell.exec('npx sequelize db:drop')
```

Comment:
I added this line, as I was only able to run the `it exists` test once and a failure would occur the next run to the unique constraint on query.